### PR TITLE
BOLT 11: Copy edit

### PR DESCRIPTION
**Pull request by toadlyBroodle**
- clarify purpose of stylesheet
- add spellchecker execution instructions to stylesheet
- bring BOLT 11 inline with stylesheet guidelines

Whew...finally done editing all the BOLTs! Won't mess with them anymore, promise.

### CODE REVIEW

File: 11-payment-encoding.md [hunk 1]
```diff
@@ -26,22 +26,26 @@ of Lightning invoices.
...
 ## Requirements
 
-A writer MUST encode the payment request in Bech32 as specified in
-BIP-0173, with the exception that the Bech32 string MAY be longer than
-the 90 characters specified there. A reader MUST parse the address as
-Bech32 as specified in BIP-0173 (also without the character limit),
-and MUST fail if the checksum is incorrect.
+A writer: 
+  - unless the Bech32 string MAY be longer than the 90 characters specified in BIP-0173:
+    - MUST encode the payment request in Bech32 (see BIP-0173).
```
Comment (rustyrussell) [inline 11-payment-encoding.md]: This should read:
```
A writer:
   - MUST encode the payment request in Bech32 (see BIP-0173)
   - MAY exceed the 90-character limit specified in BIP-0173.
```

File: 11-payment-encoding.md [hunk 2]
```diff
@@ -55,23 +59,25 @@ The following `multiplier` letters are defined:
...
-  - Otherwise:
-	- MUST fail if `amount` contains a non-digit or is followed by
-      anything except a `multiplier` in the table above
-    - If the `multiplier` is present:
-	  - MUST multiply `amount` by the `multiplier` value to derive the
-        amount required for payment
+  - if it does not understand the `prefix`:
+    - MUST fail the payment. 
+  - if the `amount` is empty:
+	  - SHOULD indicate that amount is unspecified [FIXME: indicate where or to whom?].
```
Comment (rustyrussell) [inline 11-payment-encoding.md]: - SHOULD indicate to the payer that amount is unspecified

File: 11-payment-encoding.md (hunk 1 above)
Comment (toadlyBroodle) [reply 11-payment-encoding.md]: Done...

File: 11-payment-encoding.md (hunk 2 above)
Comment (toadlyBroodle) [reply 11-payment-encoding.md]: ...and done. No worries, you're a busy man!

File: 11-payment-encoding.md [hunk 3]
```diff
@@ -55,23 +59,25 @@ The following `multiplier` letters are defined:
...
-  - MUST encode `prefix` using the currency it requires for successful payment
-  - If it requires a specific minimum amount for successful payment:
-	- MUST include that `amount`
-	- MUST encode `amount` as a positive decimal integer with no leading zeroes
-	- SHOULD use the shortest representation possible by using the largest
-	  multiplier or omitting the multiplier
+  - MUST encode `prefix` using the currency required for successful payment.
+  - if a specific minimum `amount` is required for successful payment:
+	  - MUST include that `amount`.
+	- MUST encode `amount` as a positive decimal integer with no leading 0s.
```
Comment (cdecker) [inline 11-payment-encoding.md]:
```suggestion
	- MUST encode `amount` as a positive decimal integer with no leading zeoes (`0`).
```

File: 11-payment-encoding.md [hunk 4]
```diff
@@ -137,111 +143,108 @@ Currently defined tagged fields are:
...
+  - if there is NOT a public channel associated with its public key:
+    - MUST include at least one `r` field.
+      - `r` field MUST contain one or more ordered entries, indicating the forward route from
+      a public node to the final destination.
+        - Note: for each entry, the `pubkey` is the node ID of the start of the channel;
+        `short_channel_id` is the short channel ID field to identify the channel; and
+        `fee_base_msat`, `fee_proportional_millionths`, and `cltv_expiry_delta` are as
+        specified in [BOLT #7](07-routing-gossip.md#the-channel_update-message).
+    - MAY include more than one `r` field to provide multiple routing options.
+  - MUST pad field data to a multiple of 5 bits, using 0s.
```
Comment (cdecker) [inline 11-payment-encoding.md]:
```suggestion
  - MUST pad field data to a multiple of 5 bits, using zeroes (`0`).
```

File: 11-payment-encoding.md [hunk 5]
```diff
@@ -55,23 +59,25 @@ The following `multiplier` letters are defined:
...
-  - MUST encode `prefix` using the currency it requires for successful payment
-  - If it requires a specific minimum amount for successful payment:
-	- MUST include that `amount`
-	- MUST encode `amount` as a positive decimal integer with no leading 0s
-	- SHOULD use the shortest representation possible by using the largest
-	  multiplier or omitting the multiplier
+  - MUST encode `prefix` using the currency required for successful payment.
+  - if a specific minimum `amount` is required for successful payment:
+	  - MUST include that `amount`.
+	- MUST encode `amount` as a positive decimal integer with no leading 0s.
```
Comment (cdecker) [inline 11-payment-encoding.md]: Seems we are mixing tabs and spaces here, could we make this consistent while we're here?

File: 11-payment-encoding.md (hunk 3 above)
Comment (alexbosworth) [reply 11-payment-encoding.md]: what are zeoes?

Comment (alexbosworth) [reply 11-payment-encoding.md]:
```suggestion
	- MUST encode `amount` as a positive decimal integer with no leading zeroes.
```

File: .copy-edit-stylesheet-checklist.md [hunk 6]
```diff
@@ -1,9 +1,11 @@
 # Basic checklist/stylesheet used for copy editing BOLTs
 
-This checklist/stylesheet is temporarily included here for reference purposes.
+Contributions should comply with this checklist/stylsheet to maintain correct, clear, consistent, and concise BOLTS.
```
Comment (niftynei) [inline .copy-edit-stylesheet-checklist.md]: stylsheet -> stylesheet

**Pull request merged**